### ISSUE TITLE
Null check operator used on a null value while parsing PDF

### ISSUE THREAD

**Issue by garrettApproachableGeek** [DavBfr/dart_pdf#1030]
**Describe the bug**

Using load/parse with this PDF throws this error
```
Unhandled Exception: Null check operator used on a null value
#0      PdfDocumentParser.mergeDocument (package:pdf_crypto/src/parser/document.dart:142:59)
#1      new PdfDocument.load (package:pdf/src/pdf/document.dart:107:11)
#2      new Document.load (package:pdf/src/widgets/document.dart:81:31)
#3      PdfToImage.parsePdfToImage (package:X/utils/pdf_to_image.dart:37:31)
#4      _PdfDrawingDialogState.selectPdfFiles (package:X/components/dialogs/pdf_drawing_dialog.dart:432:49)
<asynchronous suspension>
#5      _PdfDrawingDialogState.uploadPdfDrawing.<anonymous closure> (package:X/components/dialogs/pdf_drawing_dialog.dart:332:15)
<asynchronous suspension>
```

This seems to be the offending null check
`final mb = _resolve<PdfArray>(page.page['/MediaBox']!);`

**To Reproduce**
Code snippet to reproduce the behavior:
```dart
final pdf = pw.Document.load(
  pdf_crypto.PdfDocumentParser(bytes),
);
```

**Expected behavior**

PDF can be parsed without null check throwing that value is null

**Screenshots**

**Flutter Doctor**

```
[✓] Flutter (Channel stable, 2.10.2, on macOS 12.2.1 21D62 darwin-arm, locale en-US)
    • Flutter version 2.10.2 at /Users/user/fvm/versions/2.10.2
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 097d3313d8 (3 months ago), 2022-02-18 19:33:08 -0600
    • Engine revision a83ed0e5e3
    • Dart version 2.16.1
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    • Android SDK at /Users/user/Library/Android/sdk
    • Platform android-32, build-tools 32.1.0-rc1
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7772763)

[✓] VS Code (version 1.67.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.40.0

[✓] Connected device (3 available)
    • iPad (mobile) • 0000-XXX • ios            • iOS 15.3.1 19D52
    • macOS (desktop)         • macos                     • darwin-arm64   • macOS 12.2.1 21D62 darwin-arm
    • Chrome (web)            • chrome                    • web-javascript • Google Chrome 101.0.4951.54

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!
```

**Desktop (please complete the following information):**
 - [x] iOS
 - [x] Android
 - [x] Browser
 - [ ] Windows
 - [ ] Linux

**Smartphone (please complete the following information):**
 - Device: [iPad/Mac]
 - OS: [iOS/MacOS]
 - Browser [Chrome]
 - Version [15.3.1/101.0.4951.54]

**Additional context**
This PDF was created using the Notes app on an iPad
[Test.pdf](https://github.com/DavBfr/dart_pdf/files/8655594/Test.pdf)

**Comment by DavBfr** [comment]
Ok, I'll fix that.

**Comment by DavBfr** [comment]
Fixed in pdf_crypto 3.2.4

**Comment by garrettApproachableGeek** [comment]
Is there a way to tell what is released? As it seems the currently released version is 3.2.3, but the Changelog shows 3.2.4. I am guessing it didn't get deployed.
